examples/cargo/BUILD

```diff
@@ -0,0 +1,33 @@
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_binary",
+    "rust_library",
+    "rust_test",
+)
+load(
+    "@io_bazel_rules_rust//cargo:cargo_build_script.bzl",
+    "cargo_build_script_run",
+)
+
+rust_binary(
+    name = "build_script",
+    srcs = ["build_script.rs"],
+)
+
+cargo_build_script_run(
+    name = "build_script_run",
+    srcs = ["lib.rs"],
```

Comment:
Is "lib.rs" actually used by the build script anywhere? Seems kinda like a superfluous dependency to me.

Reply:
Oh right this was meant to test the access, Removed.